client/styles/main.scss

```diff
@@ -523,6 +523,7 @@ span.offline-count.diff {
     color: #999999;
     padding-top: 7px;
     padding-bottom: 7px;
+    user-select: none;
```

Comment:
This will make it impossible to select the title to copy/search it.

Reply:
That's intentional, because double clicks also mark text. However, thinking about it, why isn't `event.preventDefault()` preventing that? I'll look into this whether there's a better solution, but first let us decide what to do with the delay (see below).

Reply:
Yes, I know, I usually do double click to select words.